src/util/sss_sockets.c

```diff
@@ -374,7 +376,8 @@ static void sssd_async_socket_init_done(struct tevent_req *subreq)
     talloc_zfree(subreq);
     if (ret != EOK) {
         DEBUG(SSSDBG_CRIT_FAILURE,
-              "sdap_async_sys_connect request failed: [%d]: %s.\n",
+              "sdap_async_sys_connect request failed: [%d]: %s "
```

Comment:
`sssd_async_connect_recv()` doesn't always fail due to a timeout, so I think that the debug message should differ depending on the error.